DataPipeline-Integration-Orc-Upgrade-Libraries/build.gradle

```diff
@@ -0,0 +1,50 @@
+plugins {
+    id 'java'
+    id 'eclipse'
+}
+
+repositories {
+    //mavenLocal()
+    jcenter()
+    maven {
+        url = 'https://maven.twttr.com/'
+    }
+    maven {
+        url = 'https://jitpack.io'
+    }
+    maven {
+        url = 'https://maven.northconcepts.com/public/repositories/datapipeline'
+    }
+}
+
+sourceCompatibility = JavaVersion.VERSION_21
+targetCompatibility = JavaVersion.VERSION_21
```

Comment:
Can we use the minimum version for this demo if it's lower than 21?

Reply:
Updated version to 17.